src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -68,6 +69,21 @@ public partial class DynamoView : Window, IDisposable
         private ShortcutToolbar shortcutBar;
         private bool loaded = false;
 
+        // list of items in the right side bar. 
+        private ObservableCollection<TabItem> tabItems = new ObservableCollection<TabItem>();
+        
+        private ObservableCollection<TabItem> TabItems
```

Comment:
I think you can just make this an auto implemented property
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties